clio-server/config/docker-compose.yml.ctmpl

```diff
@@ -0,0 +1,23 @@
+{{with $docker_tag := env "DOCKER_TAG"}}
```

Comment:
This is the pattern used in `gotc`, but I'm not sure if these `with` statements are worth it for variables only used once (vs. just using the `env` substitution inline).